modules/uncertainty/R/plots.R

```diff
@@ -122,61 +118,52 @@ format.plot.input <- function(plot.inputs, convert.var, trait.order=c()){
 ##' @param fontsize (optional) list with three arguments that can be set to vary the fontsize of the title, axis labels, and axis title in the sensitivity plots
 ##' @export
 ##' @return object of class ggplot
-plot.sensitivity <- function(sa.sample, sa.spline, trait,
-                             y.range = c(0,50), median.i = 4,
-                             prior.sa.sample = NULL, prior.sa.spline = NULL,
-                             fontsize = list(title = 12, axis = 8),
-                             linesize = 1,
-                             dotsize = 2) {
+plot.sensitivity <- function(sa.sample, sa.spline, trait, y.range = c(0, 50), median.i = 4, 
+                             prior.sa.sample = NULL, prior.sa.spline = NULL, 
+                             fontsize = list(title = 12, axis = 8), 
+                             linesize = 1, dotsize = 2) {
   LENGTH_OUT <- 1000
   
   units <- trait.lookup(trait)$units
   saplot <- ggplot()
-
-  post.x <- seq(from = min(sa.sample),
-                to = max(sa.sample),
-                length.out = LENGTH_OUT)
   
-  saplot <- saplot + 
-    ## plot spline function 
-    geom_line(aes(x,y), data = data.frame(x = post.x, y = sa.spline(post.x)), size = linesize) + 
-      ## plot points used to evaluate spline
-      geom_point(aes(x,y), data = data.frame(x = sa.sample, y = sa.spline(sa.sample)), size = dotsize) +
-                                        #indicate median with larger point
-        geom_point(aes(x,y), data = data.frame(x = sa.sample[median.i], y = sa.spline(sa.sample[median.i])), size = dotsize * 1.3) + 
-          scale_y_continuous(limits = range(pretty(y.range)), breaks = pretty(y.range, n = 3)[1:3]) +
-            theme_bw() + ggtitle(trait.lookup(trait)$figid)
-              theme(axis.text.x = element_text(size = fontsize$axis),
-                   axis.text.y = element_text(size = fontsize$axis),
-                   axis.title.x = element_text(size = fontsize$axis),
-                   axis.title.y = element_blank(),
-                   plot.title = element_text(size = fontsize$title),
-                   panel.border = element_blank())
+  post.x <- seq(from = min(sa.sample), to = max(sa.sample), length.out = LENGTH_OUT)
+  
+  saplot <- saplot + ## plot spline function
+    geom_line(aes(x, y), data = data.frame(x = post.x, y = sa.spline(post.x)), size = linesize) + 
+    ## plot points used to evaluate spline
+    geom_point(aes(x, y), data = data.frame(x = sa.sample, y = sa.spline(sa.sample)), 
+               size = dotsize) + # indicate median with larger point
+    geom_point(aes(x, y), data = data.frame(x = sa.sample[median.i], y = sa.spline(sa.sample[median.i])), 
+               size = dotsize * 1.3) + 
+    scale_y_continuous(limits = range(pretty(y.range)), 
+                       breaks = pretty(y.range, n = 3)[1:3]) + theme_bw() + ggtitle(trait.lookup(trait)$figid)
+  theme(axis.text.x = element_text(size = fontsize$axis),
```

Comment:
`theme` seems not to be connected to the plot, not due to @bpbond's changes though